### PR TITLE
feat(swc/plugin): implement dummy_with_cmt proxy

### PR DESCRIPTION
**Pull request by kwonoj**
**Description:**

Implements proxy for `Span::dummy_with_cmt` for the plugin.

**BREAKING CHANGE:**

**Related issue (if exists):**

### CODE REVIEW

File: crates/swc_common/src/syntax_pos.rs [hunk 1]
```diff
@@ -199,6 +200,10 @@ pub struct MultiSpan {
     span_labels: Vec<(Span, String)>,
 }
 
+extern "C" {
```
Comment (kdy1) [inline crates/swc_common/src/syntax_pos.rs:203]: I think we need `#[cfg]`

Comment (kwonoj) [reply crates/swc_common/src/syntax_pos.rs:203]: isn't this guarded via https://github.com/swc-project/swc/pull/4268/files#diff-f2c8fa127c4fc658c5b278fa9d6084d2b8c45070d1fc7515191d2afa981bbd8bR446?

this is same as existing, i.e https://github.com/swc-project/swc/blob/main/crates/swc_common/src/syntax_pos/hygiene.rs#L78=

Comment (kdy1) [reply crates/swc_common/src/syntax_pos.rs:203]: Ah, I see, thanks!

**Review by kdy1 — APPROVED** [review]
Thanks!

---


swc-bump:
 - swc_common
 - swc_plugin_runner

**Review by kdy1 — APPROVED** [review]

**Pull request merged**